public-ui/src/app/views/ehr-view/ehr-view.component.html

```diff
@@ -33,9 +33,17 @@ <h5 id="domain-summary" class="secondary-display">
             <div class="results-grid" *ngIf="items.length > 0">
               <div class="domain-info-layout">
                 <span>
-                  <h5 id="domain-name" class="primary-display">
-                    Showing top {{items.length}}
-                  </h5>
+                  <ng-container *ngIf="totalResults <= searchRequest.maxResults; else show_pages">
```

Comment:
if the search is less than max results don't do anything special else format the numbers like (1-50 out of 26,000)